src/app/views/home/feed.ts

```diff
@@ -154,6 +154,12 @@ export default class RouteActivityFeed extends BaseRouteComponent {
 			fromCache
 		);
 
+		// Stating that we should allow follow buttons
+		// and menus to appear on the home feed.
+		if (this.feed) {
+			this.feed.shouldShowFollow = true;
```

Comment:
You can actually put this in the above object, like so:

```
this.feed = ActivityFeedService.routed(
			this.feed,
			{
				type: 'EventItem',
				url: getFetchUrl(this.$route),
				hideCommunity: true,
				hideCommunityChannel: this.channel !== 'featured' && this.channel !== 'all',
				shouldShowFollow: true,
				notificationWatermark: $payload.unreadWatermark,
			},
			$payload.items,
			fromCache
		);
```

That function takes the feed configuration. Also, I don't think you need a comment since the variable name is pretty descriptive.